suite-native/app/src/navigation/RootStackNavigator.tsx

```diff
@@ -22,21 +23,25 @@ export const RootStackNavigator = () => {
     return (
         <RootStack.Navigator
             initialRouteName={
-                isOnboardingFinished ? RootStackRoutes.App : RootStackRoutes.Onboarding
+                isOnboardingFinished ? RootStackRoutes.AppTabs : RootStackRoutes.OnboardingStack
             }
             screenOptions={stackNavigationOptionsConfig}
         >
             {!isOnboardingFinished && (
```

Comment:
Why is `if` there? I think it should work same without it because of definition in `initialRouteName`